workflows/genomeassembly.nf

```diff
@@ -6,18 +6,33 @@
 
 def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params)
 
+// Validate input parameters
+WorkflowGenomeassembly.initialise(params, log)
+
+// TODO nf-core: Add all file path parameters for the pipeline to the list below
 // Check input path parameters to see if they exist
-def checkPathParamList = [ params.input ]
+def checkPathParamList = [ params.input, params.multiqc_config, params.fasta ]
```

Comment:
You test for `params.fasta` both here and in `main.nf` but I don't even see a params.fasta in your input. I am confused about this.